.github/workflows/cifuzz.yml

```
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
```

Can you please restrict the default permissions to `read-all`? This will ensure we give no write permissions to the `GITHUB_TOKEN` generated for the workflow runs.

```suggestion

permissions: read-all

jobs:
```